src/routes/(console)/project-[project]/databases/database-[database]/delete.svelte

```diff
@@ -30,20 +97,120 @@
+                        </TableHeader>
+                        <TableBody>
+                            {#each collectionItems as collection}
+                                <TableRow>
+                                    <TableCell width={columnWidth} title="Collection"
+                                        >{collection.name}</TableCell>
+                                    <TableCell width={columnWidth} title="Document Count"
+                                        >{collection.count}</TableCell>
+                                    <TableCell width={columnWidth} title="Last Updated"
+                                        >{toLocaleDate(collection.updatedAt)}</TableCell>
+                                </TableRow>
+                            {/each}
+                        </TableBody>
+                    </TableScroll>
+
+                    {#if collectionItems.length < collections.total}
+                        <div class="u-flex u-gap-16 u-cross-center">
+                            <button class="u-underline" on:click={listCollections} type="button">
+                                Show more
+                            </button>
+
+                            {#if isLoadingDocumentsCount}
+                                <div class="loader is-small" />
+                            {/if}
+                        </div>
+                    {:else}
+                        <button
+                            class="u-underline"
+                            on:click={() => {
+                                collectionItems = collectionItems.slice(0, 3);
+                            }}
+                            type="button">
+                            Show less
+                        </button>
+                    {/if}
+                </div>
+
+                <FormList>
+                    <InputText
+                        label={`Confirm the database name to continue`}
+                        placeholder="Enter {$database.name} to continue"
+                        id="database-name"
+                        required
+                        bind:value={databaseName} />
+                </FormList>
+            {:else}
+                <p class="text" data-private>
+                    Are you sure you want to delete <b>{$database.name}</b>?
+                </p>
+            {/if}
+        {/await}
+
+        <svelte:fragment slot="footer">
+            <Button text on:click={() => (showDelete = false)}>Cancel</Button>
+            <Button
+                secondary
+                submit
+                disabled={isLoadingDocumentsCount ||
+                    (collections.total > 0 &&
+                        !error &&
+                        (!databaseName || databaseName !== $database.name))}>
+                Delete
+            </Button>
+        </svelte:fragment>
+    </Modal>
+</div>
+
+<style>
+    :global(.max-height-dialog dialog) {
+        max-height: 650px;
+    }
```

Comment:
why do we need that?

Reply:
the collections list can increase and that would increase the modal height too iirc.